index.html

```diff
@@ -241,7 +261,7 @@ <h4>Skills</h4>
 
     </section>
     <section class="contact-me flex">
-        <div>
+        <div class="contact-me-container flex">
             <div class="contact-me-items flex">
                 <div class="contact-me-heeder flex">
                     <h2>Contact Me</h2>
```

Comment:
- [ ] [optional] Please make sure the about section and contact section are in the center on all screens over 768px.
 - As I suggested above You should favor units that are percent base (vh, vw, %) over fixed units (px) so they design will be same for all screens.

### Your design

![image](https://user-images.githubusercontent.com/83803414/159944749-0b206429-ba8d-4125-9084-7a4128eb773e.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/159944811-ee9c7d2d-c466-45ec-b262-20f789b6d07f.png)